doc/getting_started/getting_started.rst

```diff
@@ -236,12 +279,12 @@ natively 64bit.
 
 To compile and run an application in this way, type:
 
-.. code-block:: console
-
-   $ cd $ZEPHYR_BASE/samples/hello_world
-   $ mkdir build && cd build
-   $ cmake -DBOARD=native_posix ..
-   $ make
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :generator: make
```

Comment:
Why not move to ninja here? Does it not work? (Just curious, not objecting)

Reply:
It does work, will change it.